tests/e2e/cucumber/features/smoke/admin-settings/groups.ocis.feature

```diff
@@ -39,3 +39,17 @@ Feature: groups management
       | security |
       | finance  |
     And "Admin" logs out
+
+  Scenario: edit groups
+    Given "Admin" creates following user using API
+      | id    |
+      | Alice |
+    Given "Admin" creates following groups using API
+      | id    |
+      | sales |
+    When "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the groups management page
+    When "Admin" changes displayName to "it sales department" for group "sales" using the sidebar panel
+    And "Admin" logs out
```

Comment:
There is a confirmation step missing which doesn't exist on other tests as well.